src/esportsbot/cogs/DefaultRoleCog.py

```diff
@@ -11,62 +11,71 @@ def __init__(self, bot):
         self.STRINGS = bot.STRINGS["default_role"]
 
     @commands.command(
-        name="setdefaultrole",
-        usage="<role_id> or <@role>",
-        help="Sets the role that the server gives to members when they join the server"
+        name="setdefaultroles",
+        usage="<@role> <@role> <@role> ...",
+        help="Sets the roles that the server gives to members when they join the server"
     )
     @commands.has_permissions(administrator=True)
-    async def setdefaultrole(self, ctx, given_role_id=None):
-        cleaned_role_id = role_id_from_mention(given_role_id) if given_role_id else False
-        if cleaned_role_id:
-            guild = DBGatewayActions().get(Guild_info, guild_id=ctx.author.guild.id)
-            guild.default_role_id = cleaned_role_id
-            DBGatewayActions().update(guild)
-            await ctx.channel.send(self.STRINGS['default_role_set'].format(role_id=cleaned_role_id))
-            default_role = ctx.author.guild.get_role(cleaned_role_id)
-            await send_to_log_channel(
-                self,
-                ctx.author.guild.id,
-                self.STRINGS['default_role_set_log'].format(author=ctx.author.mention,
-                                                            role_mention=default_role.mention)
-            )
+    async def setdefaultroles(self, ctx, *, args: str):
```

Comment:
If you wanted you could make use of the discord.Context.message.role_mentions, as it will give every role in the message as a Role object?